src/test/java/org/testeditor/fixture/commons/date/generate/DateGeneratorTest.java

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2018 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+
+package org.testeditor.fixture.commons.date.generate;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+public class DateGeneratorTest {
```

Comment:
What about
* an offset of zero?
* an offset of more than a month [year]?
* different format strings?
    * … particularly _illegal_ format strings?

Since the body of each test method will probably be almost entirely the same for each of those tests (as well as the two already existing ones), a [parameterized setup](https://github.com/junit-team/junit4/wiki/parameterized-tests) could be considered. Or do you think that's overkill?